docs/introduction.rst

```diff
@@ -19,42 +31,82 @@ can be open source or proprietary (although some distribution services, like
-applications on Linux. First and foremost, Flatpak allows a single application
-build to be installed and run on virtually any Linux distribution. It can also
-be used in combination with `Flathub <https://flathub.org/>`_, a centralized
-service for distributing applications on all distributions. This makes it
-possible for application developers to target the entire Linux desktop market
-from one place.
+Flatpak has some major advantages over most system package managers:
+
+- **Universality**: Flatpak allows applications to be installed and run on virtually any Linux
+  distribution. This includes non-GNU distributions, systemd-free distributions,
+  distributions with a read-only operating system (OS), and systems using a different architecture.
+- **Space for innovations**: Flatpak facilitates distribution maintainers to focus on their goals
+  to innovate their distribution.
+- **Stability**: Breakage in a Flatpak application will not risk the system from breaking.
+  This is because Flatpak applications and runtimes are contained to not interfere
+  with the system altogether.
+- **Rootless install**: Elevated privileges are not required when installing a Flatpak
+  application or a runtime.
 
-Flatpak also offers native integration for the main Linux desktops, so that
-users can easily browse, install, run and use Flatpak applications through
-their existing desktop environment and tools.
+Flatpak has some major advantages over other universal approaches to distributing
+applications on Linux:
+
+- **Centralized store**: It can also be used in combination with `Flathub <https://flathub.org/>`_,
+  a centralized service for distributing applications on all distributions. This
+  makes it possible for application developers to target the entire Linux desktop
+  market from one place.
+- **Decentralized by design**: While Flatpak does provide a centralized service for distributing
+  applications, it also allows decentralized hosting and distribution, so that
+  application developers or downstreams can host their own applications and
+  application repositories.
+- **Desktop integration**: Flatpak also offers native integration for the main Linux desktops, so that
+  users can easily browse, install, run and use Flatpak applications through
+  their existing desktop environment and tools.
+- **Space efficiency**: Flatpak links identical runtime and BaseApp versions to save megabytes
+  or even gigabytes worth of storage depending on the amount of applications installed.
 
 Other benefits for developers include:
 
-- **Forward-compatibility:** the same Flatpak can be run on different versions
+- **Forward-compatibility**: The same Flatpak application can be run on different versions
   of the same distribution, including versions that haven't been released
   yet. This doesn't require any changes or management by application developers.
-- **Maintained platforms:** called runtimes, these contain collections of
```

Comment:
Why is this being deleted?

Reply:
I moved it in the "Terminology" section. I'll put it back.